lib/galaxy/dependencies/pinned-lint-requirements.txt

```
@@ -5,6 +5,6 @@ importlib-metadata==4.2.0
 mccabe==0.7.0
 pycodestyle==2.9.1
 pyflakes==2.5.0
-ruff==0.0.228
+ruff==0.0.230
```

This should save you from the new Python linting errors.

```suggestion
ruff==0.0.228
```